recipes/tapestry/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version = "0.9.1" %}
+
+package:
+  name: tapestry
+  version: {{ version }}
+
+source:
+  url: https://github.com/johnomics/tapestry/archive/v{{ version}}-beta.tar.gz
+  sha256: a3db095fc2c2af8396ca51512ef36791c381c079aa616667f5ddf981d4297d81
+
+build:
+  number: 0
+  noarch: python
+
+requirements:
+  host:
+    - python >=3
+  run:
+    - python >=3
+    - minimap2
+    - samtools
+    - biopython
+    - intervaltree
+    - jinja2
+    - networkx
+    - numpy
+    - pandas
+    - plumbum
+    - pysam
+    - scikit-learn >=0.2.0
+    - sqlalchemy
+    - tqdm
+
+test:
+  commands:
+    - weave -h &> /dev/null
+    - clean -h &> /dev/null
```

Comment:
Do these import the module? If so that's fine, otherwise add an import test too.

Reply:
Yes, weave imports the module.